back/apps/core-fca/src/controllers/oidc-client.controller.ts

```diff
@@ -275,6 +276,21 @@ export class OidcClientController {
     @UserSessionDecorator(GetOidcCallbackSessionDto)
     userSession: ISessionService<UserSession>,
   ) {
+    if (req.query['error']) {
```

Comment:
It might be a good idea to use Nest's `@Query` decorator instead of `req.query`.

Reply:
Done.